buildenv

```diff
@@ -44,6 +44,20 @@ zopen_install() {
   rm -rf ${ZOPEN_INSTALL_DIR}
   mkdir -p ${ZOPEN_INSTALL_DIR}
   cp -pR out/* ${ZOPEN_INSTALL_DIR}
+  # Need to create a link from ~/zopen/usr/local/bin/gn to ~/zopen/usr/local/zopen/gn/gn/gn
+  # The link needs to be done via:
+  #   cd ~/zopen/usr/local/bin
+  #   ln -sf ../zopen/gn/gn/gn gn
+  # Potential ways to get to proper dir:
+  #   ZOPEN_PKGINSTALL: ~/zopen/usr/local/zopen
+  #   ZOPEN_INSTALL_DIR: ~/zopen/usr/local/zopen/gn/gn-main
+  # echo "ZOPEN_PKGINSTALL: ${ZOPEN_PKGINSTALL}"
+  # echo "ZOPEN_INSTALL_DIR: ${ZOPEN_INSTALL_DIR}"
+  cd ${ZOPEN_PKGINSTALL}
+  cd ../bin
+  echo $(pwd)
+  ln -sf ../zopen/gn/gn/gn gn
+  cd -
```

Comment:
Suggest:
```
pushd ${ZOPEN_PKGINSTALL}/../bin
ln -sf ../zopen/gn/gn/gn gn
popd
```
If you need to get back to the starting dir, and no longer need to pwd.
Also suggest removing the comments from line 48 to 55, as the code below them shows that.

Reply:
The scripts run in Bourne shell where pushd and popd aren't available.

Reply:
Yes, currently zopen build runs with /bin/sh which doesnt support pushd/popd.